tasks/train.py

```diff
@@ -73,6 +76,7 @@ def train(args):
 
             loss.backward()
             opt.step()
+            scheduler.step()
```

Comment:
Do we need to implement the gradient clipping? Like below from the paper

Before updating the parameters, gradients are clipped with
respect to the global gradient norm (i.e., clip by global norm)
with a maximum norm value of 1.0

Reply:
Good callout, I added gradient clipping to 1